src/gsAssembler/gsAssembler.h

```diff
@@ -495,28 +495,28 @@ class gsAssembler
 
     /// @brief Triggers computation of the Dirichlet dofs
     /// \param[in] unk the considered unknown
-    void computeDirichletDofs(int unk = 0);
+    void computeDirichletDofs(index_t unk = 0);
```

Comment:
The variable frequently called "unk" in this file is either int, or short_t, but needs not be index_t. it is usually below 10 and there is sometimes a special value -1.